roles/jenkins_builder/files/id_rsa.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAgEAtKdhpALPqLAThPbPWe7xRDd5Nv45loeri5bFNG9RnjuX7Xhi0cmgXGfUfyIIAAlm0U0bOKtGaKsT6aiBPfQ/P4EzIsrnmsm7MHdxYQMzQK5lugI0fz77CViUxTzoOdwlT0np0H4uNCmWipUtP52WaR9dGgYdktlxKR8jbP1nzqinIJ0OD/YXPUM+6a3miKEQtUl6GRWgXElBZlTwbSmqWjNWe22dQUgsGtINS0CnRSU8tVeuXjayKf41CJme89t1XhFYIbZAfrPmObOR1f3uzSTRtAF6Ph1yfjZtY1U+jU/AGvUg5umQqdUH/lKvL6WKyuvlMckHAgbw/T81pWLP9XnWYfwiWkaQtvCeG2uW0N9pmuxWjvhFBWSmJzzzsglWlQ0l6M6SaIq3Qqn5RaKWccR6xTIpu92qvRMokICcLkvH0uqqSaPAS/kqIjRD5LsoxOymJBdLdGg9BOHYClLthV6wkpuUl2voWVswLQk4E0fa4n3Q2kkg5qwMZhr1oZzDNbnOS70PLWHtCqtLOCH/GMR6gbASbPc2p3ZmtTWqwAcAp008Ct7lZD5OVkX3jYNZdJadYFlXT9c1jb0lSN3NGkmpzE/edyjj8S3iV2S48Bz7N9UBghXfmHEdu6tCXp24xgeBv7QS69Q+6U9DwjBH76qq42iJtTNrjOFyXDhkGQM= Shared Jenkins Key
```

Comment:
Can you call this file jenkins_key.pub?

Reply:
You've got both files now. Can you `git rm roles/jenkins_builder/files/id_rsa.pub`